core/src/main/scala/kafka/common/AuthorizationException.scala

```diff
@@ -21,4 +21,5 @@ package kafka.common
  * @param message
  */
 class AuthorizationException(message: String) extends RuntimeException(message) {
+  def this() = this(null)
```

Comment:
Because of this line https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/common/ErrorMapping.scala#L96 that uses reflection to create an instance of the Exception class.